frontend/src/lib/lemon-ui/LemonTable/LemonTable.scss

```
@@ -173,14 +173,14 @@
                 padding-bottom: 0.5rem;
 
                 .posthog-3000 & {
-                    padding-top: 0.3rem;
-                    padding-bottom: 0.3rem;
+                    padding-top: 0.3125rem;
+                    padding-bottom: 0.3126rem;
```

Just checking if this is a typo?
```suggestion
                    padding-bottom: 0.3125rem;
```

---

That is a typo!